### PR TITLE
add Dockerfile that deploy with docker anyway from local code base

### PR DESCRIPTION
**Pull request by orankaplan**
That enable to change local code and have the server refresh.

@omerxx this is based on your dockerfile, please review.

### CODE REVIEW

**Comment by danielhers** [comment]
@omerxx feel free to merge if you approve.

**Comment by omerxx** [comment]
Hey @orankaplan, The Dockerfile exists here in the project under '/docker' so if you commit changes please direct them to this path.
I can't really understand the change, since the container uses:
`docker run -it --restart="always" -p 80:5000 -v $PWD:/opt/anyway -w /opt/anyway omerxx/anyway-docker /bin/bash -c 'export DATABASE_URL=sqlite:////anyway/local.db && python main.py --open'
`

Hence - allowing an instant refresh whenever the code base changes..
Port forwarding and other setting are being set via the command above as well..
Can you elaborate?

**Comment by orankaplan** [comment]
- I missed the part where you add the code in your run command, it is not so clear and I prefer to have it on the Dockerfile, also, if I already cloned the repo why the docker build process need to clone it again?

- I place the Dockerfile on the root folder because of http://stackoverflow.com/questions/24537340/docker-adding-a-file-from-a-parent-directory

- I think that for the sadna usage, meaning new people (like myself) come and want to get a working env quickly it is better to build it as "plug and play" as possible, even better will be to set up a docker compose file with the ports and start command and have them just do 'docker-compose up'
@omerxx wdyt?

**Comment by omerxx** [comment]
@orankaplan:

1. The docker has to hold a working copy just for the sake of building and deploying the DB and dependencies. I encountered too many difficulties trying to do it with the attached volume which is why I preferred cloning cleanly from the remote repo to the local container. If you managed to build the env on the container without cloning then go ahead and change it accordingly.

2. Wasn't aware of this issue. The dockerfile is currently held here as backup since the docs direct users to clone a working container from docker hub. I agree this should be built on every new user's machine so let's keep it as you suggest in the main root, can you add a commit that removes `/docker`?

3. I agree. If you'd take it forward and do it, it would be awesome!

If you'd like to keep the changes for the sake of conviniency that's cool, but please add a commit that fixes the [relevant docs](https://github.com/hasadna/anyway/blob/dev/DOCKER.md).

**Comment by orankaplan** [comment]
@omerxx can you take a look, I've added the docker-compose that we discussed

**Comment by omerxx** [comment]
@orankaplan - good idea, looks great!
Can you forward port 80 to 5000 so that the app can be reached simply by ip?

File: DOCKER.md [hunk 1]
```diff
@@ -30,17 +30,17 @@ Instructions
 
 **3.** Open "Docker terminal", go to the **anyway** directory and run:
 
-    docker run -it --restart="always" -p 80:5000 -v $PWD:/opt/anyway -w /opt/anyway omerxx/anyway-docker /bin/bash -c 'export DATABASE_URL=sqlite:////anyway/local.db && python main.py --open'
-**You're all set!** Access the app on `localhost` (Linux) or (OS-X/Win) at the virtual machine’s IP (usually `192.168.99.100`); <br>
+    docker-compose up
+**You're all set!** Access the app on `localhost:5000` (Linux) or (OS-X/Win) at the virtual machine’s IP (usually `192.168.99.100`); <br>
 To retrieve a docker machine address: `docker-machine ip default` <br>
-(no need for port specification, simply an IP; e.g `192.168.99.100`)
+(no need for port specification, simply an IP; e.g `192.168.99.100:5000`)
```
Comment (omerxx) [inline DOCKER.md]: Remove `5000` according to port forwarding

**Comment by omerxx** [comment]
@orankaplan remove the port from the docs and we're good to go

**Comment by orankaplan** [comment]
@omerxx anything else needed before we can merge this PR?

**Comment by omerxx** [comment]
Sorry @orankaplan, feel free to merge it

**Comment by orankaplan** [comment]
@omerxx I don't have permission to merge

**Comment by omerxx** [comment]
Done. @orankaplan

**Comment by orankaplan** [comment]
@omerxx 👍

**Pull request merged**